docs/variants/hardkernel_odroid_h4/releases.md

```diff
@@ -1,19 +1,85 @@
-# Dasharo compatible with Hardkernel ODROID H4+
+# Hardkernel Odroid H4 Dasharo Release Notes
```

Comment:
ODROID is always in capital letters here: https://www.hardkernel.com/

Please revert all changes